NEWS.md

```diff
@@ -1,4 +1,6 @@
-# Version 1.0.9000.9133
```

Comment:
this line must not be removed I think

Reply:
There was a conflict in NEWS.md that I had to resolve manually.  I obviously botched the job.  Apologies.  I will fix.